### PR TITLE
Filter unsupported elliptic curves

### PR DESCRIPTION
**Pull request by s-vincent**
Related to #109.

### CODE REVIEW

**Comment by ereOn** [comment]
@s-vincent I made a couple of comments there too. This looks mostly good to me and I'll happily merge it once those are addressed !

**Comment by s-vincent** [comment]
@ereOn OK great, but I don't find the comments in the "Files changed" tab :(.

File: libs/fscp/include/fscp/server.hpp [hunk 1]
```diff
@@ -1255,7 +1255,7 @@ namespace fscp
 			void do_set_identity(const identity_store&, void_handler_type);
 
 		private:
-			elliptic_curve_list_type get_elliptic_curves_supported(const elliptic_curve_list_type& curves);
+			elliptic_curve_list_type get_supported_curves_elliptic(const elliptic_curve_list_type& curves);
```
Comment (ereOn) [inline libs/fscp/include/fscp/server.hpp]: You have a mismatch: `get_supported_elliptic_curves` <> `get_supported_curves_elliptic`

**Comment by ereOn** [comment]
LGTM ! Feel free to merge once you are satisfied and the CI completes !

**Pull request merged**